js/README.md

```diff
@@ -39,9 +39,8 @@ If you want, you can compile `protoc` from source instead.  To do this
 follow the instructions in [the top-level
 README](https://github.com/protocolbuffers/protobuf/blob/master/src/README.md).
 
-Once you have `protoc` compiled, you can run the tests by typing:
+Once you have `protoc` compiled, you can run the tests provided along with our project to examine whether it can run successfully. In order to do this, you shuold download the Protocol Buffer source code from the release page with the link above. Then extract the source codes and navigate to the folder named `js` containing a `package.json` file and a series of test files. In this folder, you can run the commands below to run the tests automatically.
```

Comment:
codes => code

Reply:
Hello, the spelling mistakes you mentioned are corrected. Thanks.